utils/tsdev.sh

```diff
@@ -11,9 +11,12 @@ help(){
 	echo "  -h, --help        Display this help message"
 	echo
 	echo "Commands:"
+	echo "  activate-history  Enable bash history persistence in the container"
 	echo "  build-api-cli     Build and install API and CLI clients"
 	echo "  celery            Start a celery worker"
 	echo "  logs              Follow docker container logs "
+	echo "  rebuild-dev       Stop, remove, and rebuild the dev environment (docker-compose down && docker-compose up --build -d)"
```

Comment:
Why `--build`? What is your use case for building the dev container every time?

Reply:
well it is not used every time, but when I want to rebuild, I want to rebuild everything to also cover potential changes to the docker files, dependencies or similar.

Reply:
In that case I would recommend to add it as an option/flag. But if it is too complicated, let's go with how it is. No strong feelings to be honest, I just never had to use it before 😅

Reply:
I use it all the time :-)